v2/scripts/publish_prod.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
Modified this and v1's publish_prod script to accept two version arguments, one for github release and one for package release version.

This script makes sure to include -v2 in the package names